test/extended/networking/commatrix.go

```diff
@@ -0,0 +1,82 @@
+package networking
+
+import (
+	"context"
+	"fmt"
+	"os"
+	"path"
+	"path/filepath"
+
+	g "github.com/onsi/ginkgo/v2"
+	o "github.com/onsi/gomega"
+
+	"github.com/liornoy/node-comm-lib/commatrix"
```

Comment:
any discussion made WRT a suitable host for the code ?

Reply:
we're considering moving it to openshift/library-go.
this is pointing to my own repo temporarily just to see first that the library does what we want it to do.

Reply:
or even under openshift/oc